migrations/20190503203544-create-user.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
Why should we use 'use strict' mode?

Reply:
This file was in the docs, but I guess we can remove this line of course